sdk/metric/internal/aggregate/aggregate.go

```diff
@@ -44,7 +44,11 @@ type Builder[N int64 | float64] struct {
 
 func (b Builder[N]) input(agg aggregator[N]) Measure[N] {
 	if b.Filter != nil {
-		agg = newFilter[N](agg, b.Filter)
+		fltr := b.Filter // Copy to prevent any funny stuff.
```

Comment:
```suggestion
		fltr := b.Filter // Copy to make it immutable after assignment.
```
Same as @pellared's comment from the other PR.

Reply:
I see my humor is not appreciated. :smile: